pkg/app/piped/cloudprovider/kubernetes/kubernetes.go

```diff
@@ -244,7 +244,16 @@ func (p *provider) Delete(ctx context.Context, k ResourceKey) (err error) {
 		return p.initErr
 	}
 
-	return p.kubectl.Delete(ctx, p.input.Namespace, k)
+	return p.kubectl.Delete(ctx, p.getNamespaceToRun(k), k)
+}
+
+// getNamespaceToRun returns namespace used on kubectl apply/delete commands
```

Comment:
nit nit: `.` at end of the line. (Just the convention of our team. )

Reply:
🙆‍♀️